language-examples/remedial-js/7.0-browser-rendering/README.md

```diff
@@ -0,0 +1,57 @@
+# 7.0 - Browser Rendering
+
+# Purpose
+
+# What is HTML?
+
+HTML is the standard markup language for creating Web pages.
+
+- HTML stands for Hyper Text Markup Language
+- HTML describes the structure of Web pages using markup
+- HTML elements are the building blocks of HTML pages
+- HTML elements are represented by tags
+- HTML tags label pieces of content such as "heading", "paragraph", "table", and so on
+- Browsers do not display the HTML tags, but use them to render the content of the page
+
+````HTML
+<p>Hello World</p>
```

Comment:
The goal here is to demonstrate basic concepts not to necessarily to create slides. With each example you have a sandbox to play with. So, the instructor will demonstrate the `<p>` tag, etc in the actual index.html file. The readme is more of a guide for them to follow.